autoload/linediff/differ.vim

```diff
@@ -12,8 +12,10 @@ function! linediff#differ#New(sign_name, sign_number)
         \ 'sign_number':        a:sign_number,
         \ 'sign_text':          a:sign_number.'-',
         \ 'is_blank':           1,
-        \ 'other_differ':       {},
+        \ 'other_differs':      {},
```

Comment:
Very well. I did all this basically without vimscript knowledge, only general programming knowledge. So it was mostly a mix'n'match approach ;)